src/main/cli/configs.py

```diff
@@ -39,6 +46,20 @@ class PREPROCESSING_LEVEL(Enum):
     INEFFICIENT_STATEMENTS = 4
     INT_EXPERIENCE = 5
 
+    @property
+    def __level_handlers(self):
```

Comment:
I think `level_actions` is a more proper name because 'handlers' is too general. What do you think?